zeroconf/_core.py

```diff
@@ -198,12 +198,16 @@ def datagram_received(
         self, data: bytes, addrs: Union[Tuple[str, int], Tuple[str, int, int, int]]
     ) -> None:
         assert self.transport is not None
+        _v6: Union[Tuple[()], Tuple[int, int]] = ()
```

Comment:
Why prefix with `_`?

Reply:
I'm not the original author, but my guess is cargo cult. I'll remove, and adjust the other names as you suggest.

Reply:
Trying it I see that pylint doesn't like `v6` as it's too short.

Reply:
`v6_scope_flow` seems like a name that would show intent